symengine/lib/symengine_wrapper.pyx

```diff
@@ -295,6 +295,10 @@ def sympy2symengine(a, raise_error=False):
                 return RealDouble(float(str(a)))
         ELSE:
             return RealDouble(float(str(a)))
+    elif isinstance(a, np.float16):
```

Comment:
```suggestion
    elif have_numpy and isinstance(a, np.float16):
```

Reply:
Done in: https://github.com/symengine/symengine.py/pull/352/commits/57f66c6848f54dd8dc4704f3c5e8269ed451899e